macOSUpgrade.sh

```diff
@@ -432,8 +453,21 @@ fi
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # APPLICATION
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+## Check if set install package static path to install after the OS installation is complete
+if [ -n "$installPackagePathWithInstallComplication" ];then
+    ## Since the package may be installed as side effects of events such as download_trigger defined in this script, this variables are checked at this timing.
+    if [ ! -e "$installPackagePathWithInstallComplication" ]; then
+        /bin/echo "The package does not exist in $installPackagePathWithInstallComplication. Please check your script arguments."
+        cleanExit 1
+    fi
```

Comment:
The administrator may have made an unintended configuration.
Therefore, I made it work like this.

Reply:
`startosintall` usages says `installpackage` can be specified multiple times.
```
--installpackage, the path of a package (built with productbuild(1)) to install after the OS installation is complete; this option can be specified multiple times.
```

When user set multiple pkgs in parameter 10, it will failed and catched here. But the error message does not seem to be appropriate.

Reply:
I see.
I don't familiar with the `installpackage` option. Are packages executed in order of arguments?
And did you mean it should have multiple package paths in parameter 10?

Reply:
@taniguti How about this?